src/browser/auto-update-manager.coffee

```diff
@@ -1,6 +1,7 @@
 autoUpdater = null
 _ = require 'underscore-plus'
 {EventEmitter} = require 'events'
+atom = require 'atom'
```

Comment:
I'm not sure if `atom` can be required here. The files in the `browser` directory execute in the main process, not the renderer process where most of the rest of Atom runs, including all the places where `require 'atom'` is usually used.